website/docs/r/healthcare_medtech_service.html.markdown

```diff
@@ -0,0 +1,94 @@
+---
+subcategory: "Healthcare"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_healthcare_medtech_service"
+description: |-
+  Manages a Healthcare MedTech (Internet of Medical Things) devices Service.
+---
+
+# azurerm_healthcare_medtech_service
+
+Manages a Healthcare Med Tech Service.
+
+## Example Usage
+
+```hcl
+resource "azurerm_healthcare_medtech_service" "test" {
+  name         = "tftest"
+  workspace_id = "tfex-workspace_id"
+  location     = "east us"
+
+  identity {
+    type = "SystemAssigned"
+  }
+
+  eventhub_namespace_name      = "tfex-eventhub-namespace.name"
+  eventhub_name                = "tfex-eventhub.name"
+  eventhub_consumer_group_name = "tfex-eventhub-consumer-group.name"
+
+  device_mapping = <<JSON
+{
+"templateType": "CollectionContent",
+"template": []
+}
+JSON
```

Comment:
as this is json
```suggestion
  device_mapping_json = <<JSON
{
"templateType": "CollectionContent",
"template": []
}
JSON
```

Reply:
also what are mappings? this seems like an empty colleciton?

Reply:
The device mapping is used for device data transform between IOT connector and FHIR-based resource. more information in this [docs](https://docs.microsoft.com/en-us/azure/healthcare-apis/azure-api-for-fhir/iot-fhir-portal-quickstart)

I will update the sample using a more specific json template